src/en/developer-guide/applications/controls/quicklookupsimpleedit.md

```diff
@@ -0,0 +1,127 @@
+---
+Author: Safija Hubljar
+---
+# Quick Lookup Simple Edit
+
+<p>The Quick Lookup Simple Edit control enhances the ability to edit lookup values easily. Users can:</p>
+<ul>
+  <li>Copy values with a single click on the icon</li>
+  <li>Delete records by clicking the delete icon</li>
+  <li>Open connected URLs directly by clicking the link icon</li>
+</ul>
+<p>Additionally, clicking on a field opens the editing dialog.</p>
+
+
+<img src='/.attachments/applications/Controls/quicklookupsimpleedit-1.png' width="300px" />
+<img src='/.attachments/applications/Controls/quicklookupsimpleedit.png' width="300px" />
+
+
+## Bindings
+
+<table>
+  <thead>
+    <tr>
+      <th>Property Name</th>
+      <th>Description</th>
+      <th>Of Type</th>
+      <th>Input</th>
+      <th>Output</th>
+      <th>Usage</th>
+      <th>Required</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>bindingField</td>
+      <td>A binding field to bind the control to.</td>
+      <td><code>Lookup.Simple</code></td>
+      <td>
+        <code>{ entityType: "talxis_address", </br> id: "d5fa380a-aa37-ef11-8409-0022489d8923", </br> name: "K pérovně 945/7, Hostivař, 10200 Praha 10" }</code>
+      </td>
+      <td>  
+      <code>{ entityType: "talxis_address", </br> id: "d5fa380a-aa37-ef11-8409-0022489d8923", </br> name: "K Pérovně 945/7, Hostivař, 10200 Praha 18, Czech Republic" }</code>
+        </td>
+      <td><code>bound</code></td>
+      <td><code>true</code></td>
+    </tr>
+    <tr>
+      <td>customButtonIcon</td>
+      <td>Icon of custom button.</td>
```

Comment:
Should contain information that you can only use [Fluent UI icons](https://uifabricicons.azurewebsites.net)